addon/utils/utils.js

```diff
@@ -15,7 +15,14 @@ const {
 } = Ember;
 
 export function requireModule(module) {
-  return self.requirejs.has(module) ? self.require(module).default : undefined;
+  const rjs = self.requirejs;
+
+  if (
+    (rjs.has && rjs.has(module)) ||
+    (rjs.entries[module] || rjs.entries[module + '/index'])
+  ) {
+    return self.require(module).default;
```

Comment:
no, but I'm checking if it exists before accessing it which is what the previous logic was